src/sass/components/_articles.scss

```diff
@@ -3,19 +3,50 @@
 
   &__title {
     text-align: center;
+    @include media-desktop {
+      font-size: 44px;
+      line-height: 52px;
+    }
   }
 
   &__container {
     position: relative;
+    @include media-desktop {
+      @include flex(column, center, center, nowrap);
+    }
   }
 
   &__cards {
     @include flex(column, flex-start, flex-start, nowrap);
+    position: relative;
+    @include media-desktop {
+      max-width: 740px;
+      flex-direction: row;
+      justify-content: center;
+      align-items: flex-start;
+      flex-wrap: wrap;
+      gap: 28px;
+    }
+
+    &:before {
+      content: '';
+      position: absolute;
+      background-color: $color-dark;
+      width: 110%;
+      height: 2px;
+      top: -150px;
+      left: 50%;
+      transform: translate(-50%, 50%);
+      @include media-desktop {
+        top: -182px;
+        width: 100%;
+      }
+    }
```

Comment:
can you try implementing this using `border` instead of pseudo element. I think your problem is that the border looks wider then content (`width: 110%` here), but you can achieve this by combining border with padding on container. smth like

```css
.your-container {
  padding: 0 25px;
  border-top: 2px solid $color-dark;
}
```

the less code you write to implement something the better